### ISSUE TITLE
Mode Insert pushed into mode stack again.

### ISSUE THREAD

**Issue by xaviervalarino** [brookhong/Surfingkeys#1922]
## Error details

Modes in stack: mappingsEditor,Insert,Normal

SurfingKeys: 1.12

Chrome: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:108.0) Gecko/20100101 Firefox/108.0

URL: moz-extension://aee1bc3b-c58a-4b00-9e30-6538a5dd5a74/pages/options.html

## Context

Error occurred while attempting to edit settings in Advanced mode.

**Comment by xaviervalarino** [comment]
This issue occurs when clicking on the ACE editor twice on Advanced settings page. Here is the stack trace from the console:
```
Uncaught TypeError: e.matches is not a function
    isEditable content.js:2350
    y content.js:3989
    s content.js:2852
    e content.js:2798
    addEventListener content.js:2799
    y content.js:3987
    T content.js:6228
    <anonymous> content.js:6290
    promise callback* content.js:6288
    <anonymous> content.js:6287
    <anonymous> content.js:6337
    <anonymous> content.js:6338
[content.js:formatted:2350](moz-extension://aee1bc3b-c58a-4b00-9e30-6538a5dd5a74/content.js:formatted)
    isEditable content.js:2350
    y content.js:3989
    s content.js:2852
    e content.js:2798
    (Async: EventListener.handleEvent)
    addEventListener content.js:2799
    y content.js:3987
    T content.js:6228
    <anonymous> content.js:6290
    (Async: promise callback)
    <anonymous> content.js:6288
    <anonymous> content.js:6287
    <anonymous> content.js:6337
    <anonymous> content.js:6338
```

**Comment by konfoo** [comment]
Same problem occurred in my environment.

## Error details

Modes in stack: mappingsEditor,Insert,Normal

SurfingKeys: 1.12

Chrome: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/109.0

URL: moz-extension://575a5a79-766f-46df-a10d-3cfb28cfae2d/pages/options.html

**Comment by brookhong** [comment]
Fixed in 1.13.0
